### PR TITLE
Dropdown storybook

### PR DESCRIPTION
**Pull request by evmiguel**
This PR is the Dropdown component for selecting slides. To view, run `yarn storybook`, and view the Dropdown in `localhost:6006`

### CODE REVIEW

**Comment by evmiguel** [comment]
@gnarf I'm going to change it to Semantic UI

File: src/client/package.json [hunk 1]
```diff
@@ -40,7 +40,11 @@
     "webpack-cli": "^3.3.7",
     "webpack-dev-server": "^3.8.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^0.88.1",
+    "url-loader": "^2.1.0"
```
Comment (gnarf) [inline src/client/package.json]: all 3 of these should be "devDependencies"

File: src/client/stories/button.stories.jsx [hunk 2]
```diff
@@ -20,4 +15,4 @@ storiesOf('Button', module)
                 😀 😎 👍 💯
             </span>
         </Button>
-    ));
+    ));
```
Comment (gnarf) [inline src/client/stories/button.stories.jsx]: this was the basic example from storybook yeah? we can just delete this...

**Comment by gnarf** [comment]
other than dependencies - LGTM

File: src/client/package.json (hunk 1 above)
Comment (evmiguel) [reply src/client/package.json]: Done

File: src/client/stories/button.stories.jsx (hunk 2 above)
Comment (evmiguel) [reply src/client/stories/button.stories.jsx]: Done

**Pull request merged**